packages/css/src/plugins/at-rules/parsers.ts

```diff
@@ -0,0 +1,220 @@
+import type {
+  BasicMatchInfo,
+  ComparisonOperator,
+  LengthInfo,
+  LengthUnit,
+  OperatorInfo,
+  ParsedAtRule,
+  Property,
+  PropertyInfo,
+} from './types';
+
+const REM_SIZE = 16;
```

Comment:
We know this assumption is probably wrong, even within Atlassian it's wrong in one place, but it's a good start—configurable later if it's really a problem…